cloudflare/src/handle.rs

```diff
@@ -78,7 +79,8 @@ async fn get_app_ctx(
             tracing::info!("Initialized new application context");
             Ok(Ok(app_ctx))
         }
-        Err(e) => Ok(Err(e)),
+        Err(AppCtxError::BuildError(res)) => Ok(Err(res)),
+        Err(AppCtxError::ResponseError(err)) => Err(err),
```

Comment:
We could remove the indirection in `get_app_ctx` also.

Reply:
@tusharmath did some changes, please review.